### PR TITLE
Make mountains reachable

### PR DESCRIPTION
**Pull request by jwalter**

### CODE REVIEW

**Review by ricardodevries — DISMISSED** [review]
LGTM.

**Comment by whitep4nth3r** [comment]
lol oops

Please could you add a test for this where all the other tests for "canMove etc etc"?

**Comment by whitep4nth3r** [comment]
Also city shouldn't be reachable from the forest. This was my mistake, got them mixed up.

**Comment by whitep4nth3r** [comment]
I would do it myself but I am nowhere near a computer until Monday

File: src/world/zones.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ export const PantherworldZones: _Zone[] = [
       PantherworldZoneNames.City,
```
Comment (ricardodevries) [inline src/world/zones.ts]:
```suggestion
```

**Comment by ricardodevries** [comment]
```typescript
test("Cannot move from forest zone to city zone", () => {
  const result = canMoveFromCurrentZoneToDesiredZone("forest", "city");
  expect(result).toStrictEqual(false);
});

test("Can move from forest zone to mountain zone", () => {
  const result = canMoveFromCurrentZoneToDesiredZone("forest", "mountain");
  expect(result).toStrictEqual(true);
});

test("Can move from mountain zone to forest zone", () => {
  const result = canMoveFromCurrentZoneToDesiredZone("mountain", "forest");
  expect(result).toStrictEqual(true);
});

test("Cannot move from mountain zone to beach zone", () => {
  const result = canMoveFromCurrentZoneToDesiredZone("mountain", "beach");
  expect(result).toStrictEqual(false);
});
```

**Pull request merged**